### ISSUE TITLE
Allow user to choose where to save the calendar file

### ISSUE THREAD

**Issue by mattcmerritt** [mattcmerritt/SymptomChecker#16]
Rather than just saving the user's calendar in the current working directory, it may be clearer to prompt the user about where they would like to save their calendar. We could also make the default behavior clearer by saving to the user's Documents folder and including a note about where the file was saved.

This change would likely involve adding to the ``Calendar.java`` file in the beginning of the ``saveCalendar()`` method.

**Comment by Henokk-14** [comment]
That sounds like a great idea. They could also possibly name the file itself.

**Comment by cacooper21** [comment]
I agree this would also make the user experience much simpler and easier
